### ISSUE TITLE
Entities decoding goes haywire for input values

### ISSUE THREAD

**Issue by mderijcke** [tbranyen/diffhtml#92]
Hi there,

We had an input like this:

```html
<input value="T-tag pafssageeapos;s [Month] [Year] &lt;b&gt;test&lt;/b&gt;">
```

That should have looked like:

```html
T-tag pafssagee's [Month] [Year] <b>test</b>
```

But ended up looking like:

```
T-tag pafssageeapos;s [Month] [Year] &lt;b&gt;test&lt;/b&gt;
```

DiffHTML seems to (double) encode the ampersands, which it has no business in doing. I fixed it this way:

```js
diff.addTransitionState("attributeChanged", (element, attributeName, oldValue, newValue) => {
	if (attributeName === "value") {
		$(element).val(entities.decodeHTML(newValue))
	}
})
```

But I have a strong feeling this isn't intended behavior. Maybe something to look into?

**Comment by tbranyen** [comment]
Absolutely, this is not good business. I'll fix it up and let you know once a new patch release is out.

**Comment by tbranyen** [comment]
@mderijcke can you test #93 and let me know if it fixes your issue? You can test by using npm to install that commit. if you aren't sure how:

```
npm install tbranyen/diffhtml#fix-decoding-issue
```

**Comment by tbranyen** [comment]
Fixed and published in [0.9.2](https://github.com/tbranyen/diffhtml/releases/tag/0.9.2)
